src/pyFAI/io/integration_config.py

```diff
@@ -285,7 +288,7 @@ def _patch_v4_to_v5(config):
     if "polarization_factor" in config:
         pf = config.pop("polarization_factor")
         if "polarization_offset" in config:
-            config["Polarization_description"]= PolarizationDescription(pf, config.pop("polarization_offset"))
+            config["Polarization_description"] = PolarizationDescription(pf, config.pop("polarization_offset"))
```

Comment:
Why is there a captial letter in `Polarization_description`?

Reply:
Shit, this is typo. I will have to double check all the code about it ! Thanks for spotting it.